html2text_test.go

```diff
@@ -1,11 +1,59 @@
 package html2text
 
 import (
+	"bytes"
 	"fmt"
+	"io/ioutil"
+	"path"
 	"regexp"
+	"strings"
 	"testing"
 )
 
+var (
+	destPath = "testdata"
+)
+
+func TestParseUT8(t *testing.T) {
+	html_files := []struct {
+		file                      string
+		keyword_should_not_exists string
+		keyword_should_exists     string
+	}{
+		{
+
+			"utf8.html",
+			"学习之道:美国公认学习第一书title",
+			"次世界冠军赛上，我几近疯狂",
+		},
+		{
+			"utf8_with_bom.xhtml",
+			"1892年波兰文版序言title",
+			"种新的波兰文本已成为必要",
+		},
+	}
+
+	for _, html_file := range html_files {
+		bs, err := ioutil.ReadFile(path.Join(destPath, html_file.file))
+		if err != nil {
+			t.Fatal("ReadFile  err: ", err)
+		}
+		text, err := FromReader(bytes.NewReader(bs))
+		if err != nil {
+			t.Fatal("html2Text  err: ", err)
+			t.FailNow()
```

Comment:
Remove pointless `t.FailNow()`, since it will never be reached after the `t.Fatal(...)` above.